src/test/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,51 @@
+package calculator;
+
+import calculator.separator.BasicSeparator;
+import calculator.separator.CustomSeparator;
+import calculator.separator.Separator;
+
+import javax.persistence.Basic;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+import static calculator.CalculratorValidation.*;
+
+/**
+ * <pre>
+ * calculator
+ *      CalculratorText
+ * </pre>
+ *
+ * @author YunJin Choi(zzdd1558@gmail.com)
+ * @since 2022-03-02 오전 12:35
+ */
+
+public class StringCalculator {
+
+    private int EMPTY_TEXT_ZERO = 0;
```

Comment:
상수는 `static final` 필드입니다. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private static final int EMPTY_TEXT_ZERO = 0;
```